internal/kafka/internal/services/quota/ams_quota_service.go

```diff
@@ -258,7 +261,6 @@ func (q amsQuotaService) ReserveQuota(kafka *dbapi.KafkaRequest, instanceType ty
 		AvailabilityZone("single").
 		Reserve(true).
 		Resources(&rr).
-		CloudAccountID(kafka.BillingCloudAccountId).
```

Comment:
What's the difference between a AuthorizationRequest CloudAccount and a ReservedResource Billing Marketplace Account?

Reply:
cc @pb82 @davidffrench